custom_components/tahoma/climate_devices/valve_heating_temperature_interface.py

```diff
@@ -0,0 +1,42 @@
+"""Support for ValveHeatingTemperatureInterface."""
```

Comment:
I think you accidentally committed this one, or started at a feature branch. Can you remove this file and push again? :)

Reply:
I continued from the feature branch, SomfyHeatingTemperatureInterface. That can explain it.

Reply:
Good chance that I messed up that branch before :D.